internal/accrual/api/client/client.go

```diff
@@ -0,0 +1,120 @@
+package client
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"go.uber.org/zap"
+	"io"
+	"net/http"
+	"strings"
+)
+
+const ServiceAccrualURI = "/api/orders/{number}"
+const ServiceName = "Accrual"
+
+type ErrorNoContent string
+type ErrorTooManyRequests string
+type ErrorInternalServerError string
+type ErrorUndefined struct {
+	code int
+}
+
+func (err ErrorNoContent) Error() string {
+	return fmt.Sprintf("Сервис %s вернул пустой ответ", ServiceName)
```

Comment:
В нотации  Go сообщения об ошибках должны начинаться с буквы в нижнем регистре (за исключением аббревиатур), так как если будет использоваться цепочка ошибок буква в верхнем регистре не появится в середине сообщения.
```suggestion
	return fmt.Sprintf("cервис %s вернул пустой ответ", ServiceName)
```